stix2slider/convert_cyber_observables.py

```diff
@@ -99,10 +102,10 @@ def determine_1x_object_type(c_o_object):
         else:
             return Process
     if basic_type in ["user-account"]:
-        if "extensions" in c_o_object:
+        if "extensions" in c_o_object or ("account_type" in c_o_object and c_o_object["account_type"] == "unix"):
```

Comment:
This can be simplified to `c_o_object.get("account_type") == "unix"`.

Reply:
Done